README.md

```diff
@@ -61,3 +61,21 @@ Convert a JSON representation back into an immutable object
 ### `transit.withFilter(function) => transit`
 
 Create a modified version of the transit API that deeply applies the provided filter function to all immutable collections before serialising. Can be used to exclude entries.
+
+### `transit.withRecords([recordClasses], optionalFilterFunction) => transit`
```

Comment:
I'd be tempted to leave out the filter argument, but allow `withRecords(records).withFilter(filter)` instead.